docs/contributing/Regular-Expression-Regex.html

```diff
@@ -0,0 +1,1140 @@
+---
+layout: page
+title: Regular Expression - Regex
+---
+
+<p>The purpose of this page is to provide detailed information for using Regex while editing a masterlist, as well as searching specific terms within it. Regular expressions can formally be described as follows:</p>
+
+<blockquote cite="https://en.wikipedia.org/wiki/Regular_expression">
+A regular expression (shortened as regex or regexp) is a sequence of characters that specifies a search pattern in text.
+</blockquote>
+
+<h1>General</h1>
```

Comment:
h1 renders too large:

![image](https://user-images.githubusercontent.com/2430738/178138491-3b4ca38f-635d-44e9-b17d-7c01e7e2b257.png)

Start with h3 as the largest heading in this page and work down from there.

Reply:
I've adjusted the size of the different headings in https://github.com/loot/loot.github.io/pull/109/commits/383614f05462d908cddb5afdda9a4e9fd9149a69.

Specifically I replaced `<h3></h3>` with `<h5></h5>`, `<h2></h2>` with `<h4></h4>` and `<h1></h1>` with `<h3></h3>`.